Cargo.toml

```diff
@@ -30,9 +30,9 @@ primitive-types = "0.12.1"
 proptest = "1.4"
 
 # "Internal" dependencies
-zkevm_opcode_defs = "0.150.0"
-zk_evm_abstractions = "0.150.0"
-zk_evm = "0.150.5"
+zkevm_opcode_defs = "0.151.3"
+zk_evm_abstractions = "0.151.3"
+zk_evm = "0.151.3"
```

Comment:
Along with updating the deps, you can resolve [a FIXME in the `callframe` module](https://github.com/matter-labs/vm2/blob/56cc787b578ef689b74e779c646aa70945e2c7f2/crates/vm2/src/callframe.rs#L16-L17) by importing `NEW_EVM_FRAME_MEMORY_STIPEND` from `zkevm_opcode_defs::system_params`.